pkg/registry/orasinterface.go

```diff
@@ -0,0 +1,39 @@
+package registry
+
+import (
+	"context"
+
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"oras.land/oras-go/v2"
+	"oras.land/oras-go/v2/content"
+	orasregistry "oras.land/oras-go/v2/registry"
+	"oras.land/oras-go/v2/registry/remote"
+)
+
+// OrasInterface thin layer for oras.
+type OrasInterface interface {
```

Comment:
```suggestion
type Oras interface {
```

since the type already says it's an interface and this what we expose to the outside caller